doc/user-manual/language/rewriting.lagda.rst

```diff
@@ -9,6 +9,12 @@ Rewriting
 Rewrite rules allow you to extend Agda's evaluation relation with new
 computation rules.
 
+Rules are safe to use with ```Agda.Builtin.Equality``
+if :ref:`--confluence-check <confluence-check>` is enabled.
+Confluent but non-terminating rewrite rules can not break consistency,
+unlike to non-terminating functions.
+Those results was proved `here <https://hal.science/hal-02901011v2/document>`_.
```

Comment:
Rather than "here", use author names, like "was proven by Cockx, Tabareau, and Winterhalter"